src/binder_modem.h

```diff
@@ -38,7 +38,8 @@ struct binder_modem {
     struct ofono_watch* watch;
     BinderExtSlot* ext;
     BinderData* data;
-    BinderImsReg* ims;
+    BinderIms* ims;
+    BinderImsReg* ims_reg;
```

Comment:
```
src/binder_modem.c: In function ‘binder_modem_remove’:
src/binder_modem.c:494:31: warning: passing argument 1 of ‘binder_ims_reg_unref’ from incompatible pointer type [-Wincompatible-pointer-types]
     binder_ims_reg_unref(modem->ims);
                          ~~~~~^~~~~
In file included from src/binder_modem.c:23:
src/binder_ims_reg.h:54:19: note: expected ‘BinderImsReg *’ {aka ‘struct binder_ims_reg *’} but argument is of type ‘BinderIms *’ {aka ‘struct binder_ims *’}
     BinderImsReg* ims)
     ~~~~~~~~~~~~~~^~~
```
I guess `modem->ims` needs to be replaced with `modem->ims_reg` at `binder_modem.c:494`

Reply:
Replaced with correct variable.